.gitlab/build_and_test.sh

```diff
@@ -191,6 +191,7 @@ then
     $cmake_exe \
         -C "${hostconfig_path}" \
         -DCMAKE_INSTALL_PREFIX=${install_dir} \
+        -DSUNDIALS_TEST_OUTPUT_DIR="/usr/workspace/sundials/caliper/" \
```

Comment:
```suggestion
        -DSUNDIALS_TEST_OUTPUT_DIR="/usr/workspace/sundials/caliper/examples/" \
```

Reply:
This change here is not necessary yet. I've undone them